### PR TITLE
Switch to a new BT table with Observations being an independent cache.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: store/core.go [hunk 1]
```diff
@@ -614,31 +614,116 @@ func (s *store) GetPropertyLabels(
...
+			return err
+		}
+	}
+
+	objPlaceIDNameMap := map[string]string{}
+	if len(obsDcids) > 0 {
+		// Observation DCIDs.
+		obsRowList := bigtable.RowList{}
+		for _, dcid := range obsDcids {
+			for _, pred := range []string{"0", "1"} {
```
Comment (pradh) [inline store/core.go]: Comment what is "0" and "1"?

File: store/core.go [hunk 2]
```diff
@@ -614,31 +614,116 @@ func (s *store) GetPropertyLabels(
...
-		}); err != nil {
-		return err
+		}
+		if err := bigTableReadRowsParallel(ctx, s.btTable, obsRowList,
+			func(btRow bigtable.Row) error {
+				// Extract DCID from row key.
+				parts := strings.Split(btRow.Key(), "-")
+				dcid := strings.TrimPrefix(parts[0], util.BtObsAncestorPrefix)
+				var pred string
+				if parts[1] == "0" {
```
Comment (ACscooter) [inline store/core.go]: Echoing Shanth's comment above, maybe change "0" and "1" to use constants

File: store/core.go (hunk 1 above)
Comment (Spaceenter) [reply store/core.go]: Done.

File: store/core.go (hunk 2 above)
Comment (Spaceenter) [reply store/core.go]: Done.

**Pull request merged**